source/mir/random/generic.d

```diff
@@ -0,0 +1,384 @@
+/**
+License: $(LINK2 http://boost.org/LICENSE_1_0.txt, Boost License 1.0).
+
+Authors: Sebastian Wilzbach, Ilya Yaroshenko
+
+TODO:
+- automatic method to identify inflection points?
+
+*/
+module mir.random.generic;
+
+alias f_t = double;
+
+import std.stdio;
```

Comment:
Would suggest moving this to the local place where you use it (because you'll probably want to delete it together with any `writeln` statements in the final version).